pkg/kratos/handlers.go

```diff
@@ -137,6 +144,13 @@ func (a *API) handleCreateFlow(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err != nil {
+		// Propagate the KratosErrorResponse so frontend can handle it
+		if kratosError, ok := parseGenericError(err); ok {
+			w.WriteHeader(http.StatusInternalServerError)
```

Comment:
thought: perhaps a different error code should be used. It isn't a server error and it would also make the frontend logic simpler, as they can reason about the type of error based on the http status code.

Reply:
Switched to 400 to follow the status returned by kratos